base/sort.jl

```diff
@@ -1732,7 +1764,8 @@ end
     sort!(A; dims::Integer, alg::Algorithm=defalg(A), lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward)
 
 Sort the multidimensional array `A` along dimension `dims`.
-See [`sort!`](@ref) for a description of possible keyword arguments.
+See the vector version of [`sort!`](@ref) for a description of possible keyword
```

Comment:
```suggestion
See the `::AbstractVector` method of [`sort!`](@ref) for a description of possible keyword
```

Reply:
I though of something like that but renounced because the docstrings look like this:

```
"""
    sort!(v; alg::Algorithm=defalg(v), lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward)

Sort the vector `v` in place. [...]

...
"""
    sort!(A; dims::Integer, alg::Algorithm=defalg(A), lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward)

Sort the multidimensional array `A` along dimension `dims`.
```

I guess we should either leave it like this, or add `::AbstractVector` both in the docstring text of `sort`!(A, ...)` and the docstring signature of `sort!(v, ...)`. I thought the latter would be unnecessarily technical for new users but let me know what you prefer!

Reply:
I see. That reasoning makes sense. Adding `::AbstractVector` to the docstring is not a good long term solution because of https://github.com/JuliaLang/julia/pull/46104. Perhaps "one dimensional version"?

Reply:
I changed to "one-dimensional version" as suggested.